Otherwise "make u-boot-clean" complains if arm-linux-gcc is not is your PATH.
Only needed for U-Boot 1.3.4 ??

Signed-off-by: Eric Jarrige <eric.jarrige@armadeus.org>

Index: buildroot/boot/u-boot/u-boot.mk
===================================================================
--- buildroot.orig/boot/u-boot/u-boot.mk	2011-08-08 12:13:33.000000000 +0200
+++ buildroot/boot/u-boot/u-boot.mk	2011-08-08 12:13:40.000000000 +0200
@@ -180,7 +180,8 @@
 u-boot: $(U_BOOT_TARGETS)
 
 u-boot-clean:
-	-$(MAKE) -C $(U_BOOT_DIR) clean
+	-$(MAKE) CROSS_COMPILE="$(TARGET_CROSS)" ARCH=$(U_BOOT_ARCH) -C $(U_BOOT_DIR) clean
+	rm -f $(U_BOOT_DIR)/$(U_BOOT_BIN)
 	rm -f $(MKIMAGE)
 
 u-boot-dirclean:
